common/controllers/framework/framework-step.js

```diff
@@ -134,10 +129,15 @@ class FrameworkStepController extends FormWizardController {
     try {
       // wait for all responses to resolve first
       if (responses.length) {
-        await services[assessment.framework.name].respond(
-          assessment.id,
-          responses
+        // convert "service-name" to "serviceName"
+        const serviceName = assessment.framework.name.replace(
+          /-([a-z])/g,
+          function (g) {
+            return g[1].toUpperCase()
+          }
```

Comment:
oh wow, regex and everything! If this is the only place this is used, you could just move the mapping from the previous version within this function. It's a bit clunkier, but a bit more explicit about what's happening. So something like:
```
const services = {
  'person-escort-record': req.services.personEscortRecord,
  'youth-risk-assessment': req.services.youthRiskAssessment,
}
```